tools/licenses/lib/filesystem.dart

```diff
@@ -358,6 +358,12 @@ abstract class Directory extends IoNode {
   Iterable<IoNode> get walk;
 }
 
+extension DirectoryList on Directory {
```

Comment:
Please avoid extension methods: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-extension

Reply:
It's an extension that's just used in this file. I can change the visibility to private and add a comment?

This is to get around a super odd type hierarchy in filesystem.dart.

I could also change this to a static helper method.

Reply:
I updated the call sites to use a static helper method.